src/plugins/data_explorer/public/components/sidebar/index.tsx

```diff
@@ -59,23 +74,86 @@ export const Sidebar: FC = ({ children }) => {
     }
   }, [indexPatternId, activeDataSources, dataSourceOptionList]);
 
+  const redirectToLogExplorer = useCallback(
+    (dsName: string, dsType: string) => {
+      return application.navigateToUrl(
+        `../observability-logs#/explorer?datasourceName=${dsName}&datasourceType=${dsType}`
```

Comment:
I guess we have a `application.getUrlForApp` method to get the url for a specific application

Reply:
Thanks for pointing it out. This is not the change introduced in this PR and only now serves as temporary solution which will be soon removed in upcoming release versions.

Reply:
Get it.